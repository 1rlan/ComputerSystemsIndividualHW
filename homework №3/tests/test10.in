63487134687346.13486314876
