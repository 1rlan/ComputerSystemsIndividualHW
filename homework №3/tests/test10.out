39892.866375
